tasks/sf-sns-report/package.json

```diff
@@ -21,8 +21,9 @@
     "test": "ava",
     "test-coverage": "nyc ava",
     "build": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack",
+    "build-lambda-zips": "(cd dist && rm -f lambda.zip && zip lambda.zip index.js)",
     "watch": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress -w",
-    "prepare": "npm run build"
+    "prepare": "npm run build && npm run build-lambda-zips"
```

Comment:
I had an issue with the `build-zip-lambdas` doing a release. See https://ci.earthdata.nasa.gov/browse/CUM-CBA1215-DP-10/log. Do you think this would be resolved?

Reply:
@Jkovarik should be able to fix that, I'll talk to him about it.